luthier/src/main/java/com/yahoo/bard/webservice/config/luthier/LuthierIndustrialPark.java

```diff
@@ -58,6 +60,7 @@ protected LuthierIndustrialPark(
      *
      * @return the dimension instance corresponding to this name.
      */
+    // TODO: Should this really be a package protected method?
```

Comment:
No. It should be a public method. It was just slapped together very quickly so that you'd have something to work with.